src/vs/workbench/contrib/chat/browser/chatContentParts/media/chatTerminalToolProgressPart.css

```diff
@@ -151,16 +151,21 @@
 .chat-terminal-output-container > .monaco-scrollable-element {
 	width: 100%;
 }
+.chat-terminal-output-container:focus-visible {
+	outline: 1px solid var(--vscode-focusBorder);
+	outline-offset: 2px;
+}
 .chat-terminal-output-body {
 	padding: 4px 6px;
 	max-width: 100%;
-	height: 100%;
 	box-sizing: border-box;
+	min-height: 0;
 }
-.chat-terminal-output-content {
-	display: flex;
-	flex-direction: column;
-	gap: 6px;
+.chat-terminal-output-terminal.chat-terminal-output-terminal-no-output {
+	display: none;
+}
+.chat-terminal-output-container .xterm-decoration-overview-ruler {
+	display: none !important;
 }
```

Comment:
Try using the xterm option `ITerminalOption.overviewRuler` instead. Reaching into the DOM like this is private API and could fail expectedly now or later.